doc/rls_mode.md

```diff
@@ -14,39 +14,92 @@ The type of the parameter is an object with the following fields:
 
 By default, it is `null`.
 
-### Debugging
+## Setting up
 
-There is an output channel named "Rust Language Server" which is used to show messages from RLS.
+First of all, you have to download the RLS sourcs:
 
-To open it, perform the following steps:
+```bash
+git clone https://github.com/rust-lang-nursery/rls
+```
 
-* Click "View" on the menu
-* Click "Output" on submenu
-* Click on the listbox which is to the right of the shown panel
-* Choose "Rust Language Server"
+Depending on whether you have rustup or not, there are different ways you can set up this plugin.
+
+* [Setting up without rustup](#without-rustup)
+* [Setting up with rustup](#with-rustup)
+
+#### Without rustup
 
-For making RLS print more data, refer the "Debug RLS" section below.
+**Note:** You should only do this if you do not have rustup because otherwise rustup will not work anymore.
```

Comment:
Both are acceptable, though I think your suggestion is nicer. I am not an expert on some of the more technical rules of English, though I know the basics well.